discord-scripts/fix-linear-embed.ts

```diff
@@ -27,20 +31,20 @@ const processedMessages = new Map<
 >()
 
 // let us also track sent embeds to delete them if the original message is deleted or edited WIP
-const sentEmbeds = new Map<string, Message>()
+const sentEmbeds = new Map<string, Map<string, Message>>()
 
 let issueTagRegex: RegExp | null = null
 
 function initializeIssueTagRegex() {
   issueTagRegex =
-    /(?<!https:\/\/linear\.app\/[a-zA-Z0-9-]+\/issue\/)[A-Z]{3,}-\d+\b/gi
+    /(?<!https:\/\/linear\.app\/[a-zA-Z0-9-]+\/issue\/)(?<![<[])[A-Z]{3,}-\d+(?![>\]])\b/gi
 }
 
 const projectRegex =
   /https:\/\/linear\.app\/([a-zA-Z0-9-]+)\/project\/([a-zA-Z0-9-]+)(?:#projectUpdate-([a-zA-Z0-9]+))?/g
 
 const issueUrlRegex =
-  /https:\/\/linear\.app\/([a-zA-Z0-9-]+)\/issue\/([a-zA-Z0-9-]+)(?:.*#comment-([a-zA-Z0-9]+))?/g
+  /(?<![<[]])https:\/\/linear\.app\/([a-zA-Z0-9-]+)\/issue\/([a-zA-Z0-9-]+)(?:.*#comment-([a-zA-Z0-9]+))?(?![>\]])/g
```

Comment:
The URL in an inline URL is surrounded by () rather than [], right? Isn't this regex (and the issueTagRegex above) checking for [] surrounding? (In addition to `<>`.)